pkg/apis/messaging/v1beta1/channel_types.go

```diff
@@ -56,6 +56,8 @@ var (
 
 	// Check that we can create OwnerReferences to a Channel.
 	_ kmeta.OwnerRefable = (*Channel)(nil)
+
+	_ duckv1.KRShaped = (*Channel)(nil)
```

Comment:
I couldn't help but notice we added a comment to the serving PR that explained this check, could I be a pain and ask we keep the same level of comments here? Thanks!